klayout/drc/testing/testcases/unit/nat.yaml

```diff
@@ -0,0 +1,2 @@
+nat:
+  --variant: "C --connectivity"
```

Comment:
is that new should we also add it in https://github.com/google/globalfoundries-pdk-libs-gf180mcu_fd_pv/pull/73 ?